### ISSUE TITLE
Twitter 记录（2021-12-1）

### ISSUE THREAD

**Issue by zhong-my** [zhong-my/blog#4]
这里记录我在推特说过的且有价值的话。

以时间线的方式记录。

为了让读者更好理解，内容多少有些修改。

```
面试是一个遭受打击的过程，不经历你永远不知道自己有多菜。

2021/10/21
```
---
```
所有成功背后都有无数的努力在里面。

你每天看代码三小时，别人看五小时。

你每天输入输出三小时，别人持续五小时。

有的人天生有好脑袋，而你没有。

2021/10/21
```
---
```
一项体育运动的最佳状态应该是既能赢球又能玩得开心。

2021/10/16
```
---
```
设计架构比写代码更困难。

2021/10/14
```
---
```
发展越迅速的技术，越容易出现赢家通吃的局面。

2021/10/13
```
---
```
[去中性化]对于普通用户来说就是未来。

瓦解中心化任务的权威性，把工作任务的核心转移到每家每户手里。

2021/10/12
```
---
```
智能合约（Smart Contract）是近十年最伟大的发明之一。

2021/10/11
```
---
```
不接受主流的想法和意见，是做「研究型」工作的第一步。

2021/10/11
```
---
```
人为自己而活，为自己而死，为自己而疯狂，为自己而颠沛流离。

#做一个精致的利己主义者

2021/10/10
```
---
```
What's the language?

Language is connected with you and me.

2021/10/7
```
---
```
媒体对新技术、新现象一般有两种态度：一种是保守型，置之不理。一种是批判型，抹黑正在发生的事情。

媒体工作者对新闻报道没有绝对的公平、公正，但有立场，不要期待所谓的“正义”。

比如对「加密货币」新技术的态度，除了真正懂技术的那群人，其他人对“新”货币的态度都因为媒体的影响变得不正常。

2021/10/7
```
---
```
Proof-of-work and Proof-of-stake

关于区块链的两种不同的入场机制/共识机制。

Work 表示工作量证明，拥有大量的资源和强大的算法算出奇葩的数学题，再写入区块，成功上链，就能获得奖励。

Stake 表示资金证明，拥有足够的资金和一般的算力就能入场，做出题目写入区块获得奖励。

显然，对于普通人来说，Proof-of-work 的入场门槛更高。

2021/10/6
```
---
```
Made product to make your freedom.

2021/10/5
```
---
```
货币增值主要靠流通。

货币流通率越好，效率越高，使用的范围和场景越广，市场对货币的依赖和可信度就会越高。

随着流通率的提升，货币的价值在投资者眼里就会是块沉甸甸的“金子”。

2021/10/2
```
---
```
长期没有社交活动，表达能力都变弱了。

讲述一个想法，脑子里有一张蓝图呈现整体轮廓，这张蓝图由无数个点之间互相连接而成，最终呈现出完整的思路。

但是要把每个点说清楚，让对方听懂，却非常困难。

输入的多，输出的少。人还是要多说话，哪怕是废话。

2021/9/30
```
---
```
#cryptocurrency 

Before: CEX(Centralized Exchanges)
After: DEX(Decentralized Exchanges)

2021/9/28
```
---
```
UFC 作为一场“表演赛”，参赛双方事先通过激化矛盾来吸引关注，这种方式远远比主办方砸钱买热度上话题有效得多。

为什么说是一场“表演赛”？因为比赛的输赢是其次，更重要的是观众的欢呼声。表演者一拳打在对方脸上，高举双手，就会有无数的掌声在耳边响起，实在有趣。

2021/9/27
```
---
```
仅“招人请发邮件”这一条，这道门槛，就能过滤百分之99的HR。

招聘者和求职者之间应该是开放、包容、合作、平等的关系对待，适合自己的才是最好的。

2021/9/15
```
---
```
Context switching is one of the biggest productivity killers.

2021/9/8
```
---
```
足球教练（Ted Lasso）是一部优秀的英美电视剧节目，我给电影4·5分。

如果媒体给资本控制，而不是国家掌握，那这个媒体推出的报道必定是受利益驱使，最终变成一推垃圾。

参考BBC。

2021/9/5
```
---
```
在公共场合戴口罩远远比想象中的有作用，有益健康。

人体与空气之间要接触无数的细菌，单单一个口罩，就能抵挡大部分的口、鼻等传染疾病。

在大流行病期间，口罩可以说是最伟大的发明。

2021/9/3
```
---
```
互联网「知识付费」概念新出来的时候，大家都觉得新颖，都愿意尝试。但当这些人玩腻了、看腻了，这种概念就不再具有魅力。

新鲜事物可以火几年，但不能持续一辈子，这就是互联网发展规律。

互联网是有活力的地方，只有年轻活跃的脑袋才有资格入场。

2021/9/3
```
---
```
What is creativity?

Just takes an idea and does better.

2021/9/3
```
---
```
在互联网上你不会「英语」和「建立品牌」以及「录视频」，那么你几乎是个穷人。我称之为「互联网穷人」。

2021/8/6
```
---
```
国外小米线下商店支持虚拟货币交易。

我觉得各个国家，无论你是内有的还是外来的，只要遵守本国相关法律规定，什么能做，什么不能做，自己要有数。作为企业，努力挣钱就是首要目标。

所以，无论何种虚拟货币，中国不支持，那就不做，葡萄牙等外国国家支持，那就做。

为了挣钱，企业大多数行为都是基于法律之上，而不是道德之上。

2021/8/6
```
---
```
日本的啃老族越来越多，社会问题越来越复杂。韩国和日本在这方面高度相似，虽然都是发达国家，但是在处理问题上都不及欧美出色，约束较多。

都说东亚人的社会压力大，但是中国和这些国家不同，我们的模仿对象是美国，追求更加积极开放的社会环境。

人是社会性动物，没有社交的生活离死亡就不远了。

2021/8/5
```
---
```
TL;DR (too long, didn’t read)

2021/8/4
```
---
```
随着中国经济的进步和崛起，在中国的外国企业将会面临更多的挑战。

挑战的主要地方会是由产品问题导致的互联网舆论危机。

2021/8/2
```
---
```
不同编程语言的开发环境下，语言之间如何交互、作用，是一大开发难题。

2021/7/31
```
---
```
这个世界，在任何领域都是复杂的。

政治文化没有绝对的优劣，科学技术也没有绝对的强弱。

作为个人，只有把能控制的东西牢牢把握在自己手上，才不会随波逐流，任人宰割。

2021/7/30
```
---
```
互联网发展必然是打破信息隔阂和闭环，现在和将来，想通过信息差赚钱变得越来越不现实。

2021/7/28
```
---
```
我最喜欢的英语语法书《语法俱乐部》。

从句子的结构讲起，主语、谓语、宾语、补语。从简单句、合句到复杂句，帮助英语学习者快速掌握英语这门语言。

https://t.co/nKYbj9jmGy

2021/7/26
```
---
```
在推特说中文新闻或事情的时候最好把学历、智商和各种能证明认知能力的指标提上，不然看的围观群众还以为是一帮白痴在这里自娱自乐。

这些白痴根本不在乎事实和真相是什么，它们只在乎自己是否足够恶心，是否得到满足。一帮唯利是图、自私自利的家伙。

2021/7/25
```
---
```
我是一个没有感情的动物。

我今生最大的错误就是学习了计算机这门课。

我的人生注定是失败的。

2021/7/25
```
---
```
今天的天空属于富人。

2021/7/21
```
---
```
中文互联网的标准是什么？为什么我觉得理所应当的事情会不合理？为什么我觉得能说能做的事情变化无常？

我对中文互联网的环境太模糊了，以为是正确的却离大众渐行渐远，思想和觉悟都出现了偏差，太难理解了，我太愚昧了。

无法理解什么样的方式才是正确的方式，复杂多变。

2021/7/20
```
---
```
世界上有无数种语言，无数种声音，任何一个社交平台都应该是百花齐放，百鸟争鸣。

只有一种声音的地方是黑暗的，那不应该适合人生存。

2021/7/19
```
---
```
比起那些“做错事”被世人唾弃的人，我更心疼那些“做实事”被人遗忘的人。

2021/7/19
```
---
```
这个世界的运转方式只有两个字：利益。

人做的任何事情都有理可循，而不是凭空出现，也不是他人强加。

“自私”这个词现在是贬义词，但未来会变成褒义词。

富裕的人有理，穷苦的人不仅无理，还无处讲理。

2021/7/19
```
---
```
没有任何一个好玩的游戏不需要花钱。

2021/7/16
```
---
```
在任何时候“钱”都是首位，无论你在哪里。

很现实的话题，但在互联网上不现实。

2021/7/14
```
---
```
全世界都活得像只“狗”一样，连父母也是。

如果在一个地方工作5到10年不能买套房，这就是政府的问题、社会的问题。

真TM现代笑话。

2021/7/14
```
---
```
So be careful what you talk about because if you've never seen something, you can't judge it.

2021/7/14
```
---
```
财富自由后是否需要工作？

为什么不是精神自由后是否需要工作？

自从20岁毕业后我在家已经有一年多，期间利用互联网完善且满足自身的精神需求。这个过程减少物质需求，活得自然。通过在互联网世界享受现实世界里没有的自由。

2021/7/7
```
---
```
我总结的中文互联网环境：捧的越高，摔的越惨。

2021/7/5
```
--
```
作为个人，越感受生活的不公，越期待未来成就自己的生活。

遥远的地方，西方的光辉，是自由的，是无虑的。

2021/7/4
```
---
```
我从来不相信“成功人士”从无到有、从零到一的过程，因为这在社会上是不现实的。

所谓的“成功”是整合资源，是把不属于自己的东西凝聚起来，成就所谓的“成功”。

不要相信从“苦难”、“奋斗”到“幸福”，因为这群人在当时的社会就是那群“幸福”的人，最后实现的“幸福”只是前进一步，从那20%变成10%。

呵

例如一个有趣的例子，明星如何成名？

以为是在横店当个跑龙套的脚本演员？以为有“灰姑娘”不懈努力，最后就有王子给你穿水晶鞋？拜托，这是写给小孩子看的童话故事，这与现实严重脱轨。摆脱，你已经成年了，该懂事了。

明星成名离不开背后的资本，这是社会发展的必然。无论你讨论任何一人，了解后面的背景都必将让你大吃一惊。

如果你是社会底层的“穷苦”人民，想要改变身份？想要改变社会地位？显然，这在当下的社会都不可能。

这是一个缺乏激情、陷入平庸的社会环境，每个人都有压力，却释放不出任何动力，实在要憋死人了。

追求变化，是今夜睡梦中的理想幻境，只能在“这里”甜甜的笑一笑，却不能透露出半点声音。

2021/7/3
```

```
年轻人选择自由生活，不再就业。我觉得这不是个人的问题，而是整个社会的问题。从毕业到现在我不去工作，感受“自由自在”是多么快乐，享受生活的同时批判社会的不公，足矣。

年轻人拒绝工作，也是一种选择。

2021/7/3
```
---
```
看到一句话：在中国，人的个性无法体现出来，尽量听取他人的意见，团队合作。

挺羡慕西方崇尚的个人英雄主义，因为我也想成为英雄，自己也好，他人也罢。

2021/6/30
```
---
```
保持一项纪录，21岁之前不赚一分钱，不为社会做一点贡献。

#躺平

2021/6/29
```
---
```
TikTok作为中国人创造的东西居然不给中国人用，真是天大的笑话。

2021/6/29
```
---
```
当程序员没有“咖啡”提神醒脑的东西还真不行，日常间接性犯困，身不由己。

中国人喜欢喝茶，西方人喜欢喝咖啡。对我而言，喝茶就像饭后的甜点，有多种口味可以品尝。咖啡就像早晨的闹钟，时刻提醒自己不要犯困，味道什么的，不重要。

没有茶，可以换种东西品尝。

没有咖啡，真找不到合适的“醒脑”东西

2021/6/27
```
---
```
生产商业产品，原材料是最基础的一环，也是提升利润空间的关键一环。

把原材料的信息公开透明，是否有利于消费者知悉商品详情，提高消费和购买力？

比如我想买一个蛋糕，作为消费者，我可以知道一个蛋糕里有几个鸡蛋、奶油成分、水果份量等等信息，这些信息能否防止商家在制作过程中偷工减料？

2021/6/24
```
---
```
曾经我以为“努力”就能获得成功，赚大钱。

现在我认为“成功”是拥有大量资源和“继承”人士的玩物，只要这些人想有作为，社会上的资源永远都是他们的。

痛恨这一群人。

2021/6/19
```
---
```
我觉得，中国曾经那些一炮而红，挣快钱的日子已经一去不复返。随着法制和人情的界限越来越清晰，对一些合法和不合法的东西逐渐形成公有的约束力，那么那些“特别”、”古怪”的事情只可能短暂存在，或根本不可能存在。

以前“成功人士”能因为法律缺陷获得商业上的成功，可现在不一样了，一切都变了。

穷是病。

2021/6/19
```
---
```
Bitcoin is shit. Nobody believed it.

2021/6/12
```
---
```
美国普通大学和职业大学好像没什么不同，区别在于专业目的性，一个是综合教育，一个是技术教育。

怎么一和“这里”对比，就有了区别。

我最大的困扰是找工作，你没有读本科大学就注定一辈子的命运，永远低人一等？

明明智商差不多，却有了台阶。

高考的弊端。

考量一个人一定要有合理的标准，打破固有思维，提倡人人平等。

在互联网环境下，学历已经是过去。

2021/5/31
```
---
```
好像成功的人都有一段美好的童年故事。

2021/5/31
```
---
```
Learning English is very easy because there are only three thousand basic words.

2021/5/26
```
---
```
真羡慕周杰伦 20 岁就能有知己，21 岁出道首发个人专辑。

而我就不行。写代码的人各式各样，但成功的人却不多见。

伯乐不常有，知己不常有。

2021/5/22
```
---
```
有一个问题：普通民众的政治觉悟如何衡量？需要达到什么指标才算合格？没有接受过高素质、高等教育（指学历）的群众在没有享受「高」福利的前提下如何保证政治觉悟？

一句话说就是：凭什么你有钱有资源，而是却什么都没有？

这是个问题。

2021/5/22
```
---
```
在互联网时代，比特币永远是资本家的高级玩物。

#bitcoin

2021/5/17
```

```
经过几天练习，Go设计模式（Go Patterns）三个模块已经完成。

创造模式（creational）、结构模式（structural）、行为模式（behavioral）。

看维基百科还有个并发模式（Concurrency），有点意思。

https://github.com/zhong-my/go-patterns

2021/5/14
```
---
```
招聘工作岗位写工作经验，没有能力要求，无视能力强度不是耍流氓吗？

我建议所有招聘网站将工作经验和学历删掉，只留下岗位要求和能力证明两个选项，可以帮助企业找到合适的人才。

2021/5/13
```
---
```
最近最大的笑话，太可笑了。

以“环保”的名义换取人们的理解。明眼人都知道，难道矿机在世界各地工作会因为特斯拉这点交易记录而让自然资源变得“过剩”吗？

我猜想，未来马斯克在自己的产品上不会轻易支持比特币交易，而转为更有市场价值的狗狗币铺路（马斯克大量持有）。

2021/5/13
```
---
```
区块链有机会成为下一个十年的核心技术，「独特」和「唯一性」会成为主旋律，不能更改，不能删除，只能添加。这种对数据处理的方式真让人羡慕，没有互联网，现实中怎么实现这种事情？几乎没有可能吧？

中本聪利用区块链技术实现「点对点的电子现金系统」（A Peer to Peer Electronic Cash System），第一个设计并实现了区块链的数据库应用。

哦，世界上竟有这般天才，还是个日本人。

bitcoin.org/bitcoin.pdf

2021/5/12
```
---
```
Bitcoin is a big damn problem… more people hold on to that like a child have tools.

2021/5/11
```
---
```
I’m a poor man, so fuck all rich men.

2021/5/8
```
---
```
人在20岁真是人生中最复杂、最迷茫的阶段，想做的事不能做，不想做的事情无可奈何，原来这就是融入社会的过程。

2021/5/8
```
---
```
现如今，传统超市为何依然繁荣？

越来越多的人选择网上购物，利用互联网的便利丰富生活。但周围的超市依然存在。可能因为大多数人选择在外吃饭，而不是在家做饭。

商超缺乏有趣的东西留住人，没有吸引年轻人去的动机，又怎么赚年轻人的钱包？有趣的娱乐设施特别重要。

2021/4/21
```
---
```
我希望，在下一个世界，有这样一种变化。

所有的穷人都成为富人，所有的富人都变成穷人。

这个社会是不平等的，所以需要有人做些事情，做些疯狂的事情把那些人拉下来，让周围都为之疯狂，把世界变成适合疯子生存的世界。

稳定的环境让上面的人活得太好，却做得太少。

是时候需要改变了。

2021/4/20
```
---
```
“努力学习”没有定义，做的事情无法估量。

这个社会下生存，穷人没有空间，路上都是敌人。要做到，隐藏彼此之间的差距，和黑色下的恐怖游戏，暗暗揣摩彼此之间的心思。

每天入睡和早起时都有这种感觉，痛并快乐着。

2021/4/20
```
---
```
自从外部势力放弃我国干涉内政，社会得到稳定后，国内经济内循环和发展就成为了关键。人家没法通过军事行动击溃你，则改用经济手段打压你。这也算是一件“趣事”，好似没糖吃的隔壁小孩常常讥讽你说：“我糖比你多，也比你甜，想尝尝吗？”

2021/4/16
```
---
```
回顾历史从一无所有到科学、民主、自由、富裕、独立、自强，新文化的作用在那个时期有前所未有的作用，激发了一代代年轻人有勇气站出来，得来不易。

现在的生活，得来不易。

我觉得自己还是对科技、创新和发展有向往，对未来有期待。

2021/4/15
```
---
```
Understanding history in our country is important. What are things? what are the characters? how to do it in the future? history gives me all the answers on anything.

2021/4/14
```
---
```
Why Japanese can discharge nuclear waster into the sea? That's not fair for our human life.

Anyone likes to eat seafood, we want clean food and water, how to make health better?

2021/4/13
```
---
```
I don't know if something broke my heart. It is distinctive that changing my life, changing my work. I can't sleep too deep very well, because it touched me and say "wark up, this time is too later, you have something be doing, not time can give you thanking."

Oh, man. I don't know what you want, but I'm very tired.

2021/4/10
```
---
```
经历无数次痛苦与折磨，到最后才发现自己是天才。

- 高度心理障碍患者

2021/4/8
```
